src/coreclr/src/jit/gentree.cpp

```diff
@@ -16759,7 +16759,7 @@ GenTree* Compiler::gtGetSIMDZero(var_types simdType, var_types baseType, CORINFO
                 }
                 break;
 
-#if defined(_TARGET_XARCH4_) && defined(FEATURE_HW_INTRINSICS)
+#if defined(TARGET_XARCH) && defined(FEATURE_HW_INTRINSICS)
```

Comment:
Note: Functional change.

Reply:
Functional only for checked or debug builds, as the code in this `#if` only includes asserts.